terminal/command.go

```diff
@@ -174,7 +176,7 @@ func threads(client service.Client, args ...string) error {
 
 func thread(client service.Client, args ...string) error {
 	if len(args) == 0 {
-		return fmt.Errorf("you must specify a thread")
+		return printscope(client)
```

Comment:
Would you mind reverting that and submitting it as a separate PR? I'd like to keep this patch focused on the main feature.